### PR TITLE
v0.35 initial updates

### PR DESCRIPTION
**Pull request by dedavis6797**
- [x] New PowerToys Run plugin: VSCode workspaces
- [x] PowerToys Run options to set monitor the query is opened on
- [x] Fancy Zones custom layout hotkey option

### CODE REVIEW

**Comment by PRMerger17** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/run.md [hunk 1]
```diff
@@ -107,6 +108,16 @@ If the program plugin's option "Include in global result" is not selected, be su
 
 `.Visual Studio Code -- C:\myFolder`
 
+## Monitor Positioning
+
+If multiple monitors are in use, PowerToys Run can be launched on the desired monitor by configuring the appropriate launch behavior in the Settings menu. Options include opening in:
```
Comment (htcfreek) [inline hub/powertoys/run.md]:
```suggestion
If multiple monitors are in use, PowerToys Run can be launched on the desired monitor by configuring the appropriate launch behavior in the Settings menu. Options include opening on:
```

**Review by htcfreek — CHANGES_REQUESTED** [review]
small nit

**Comment by PRMerger17** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/run.md [hunk 2]
```diff
@@ -72,6 +72,7 @@ These default activation phrases will force PowerToys run into only targeted plu
   | `>` | Shell command only. Example `>ping localhost` to do a ping query |
   | `:` | Registry keys only. Example `:hkcu` to search for the HKEY_CURRENT_USER registry key |
   | `!` | Windows services only. Example `!alg` to search for the Application Layer Gateway service to be started or stopped |
+  | `{` | Visual Studio Code previously opened workspaces, remote machines (SSH or Codespaces) and containers. Example `{powertoys` to search for workspaces that contain 'powertoys' in their paths
```
Comment (crutkas) [inline hub/powertoys/run.md]: We should directly mention what is off by default.

File: hub/powertoys/fancyzones.md [hunk 3]
```diff
@@ -64,6 +64,12 @@ The additive **Canvas** model starts with a blank layout and supports adding zon
 
 ![FancyZones Window Editor Mode](../images/pt-fancyzones-canvaseditor.png)
 
+Once a custom layout has been created, the layout can be configured to a user-defined hotkey to quickly apply it to the desired desktop. The hotkey can be set by opening the custom layout's edit menu. Once set, the custom layout can be applied by pressing the `Win ⊞ + Ctrl + Alt + [hotkey]` binding. The layout can also be applied by pressing the hotkey when dragging a window.
```
Comment (crutkas) [inline hub/powertoys/fancyzones.md]: Make this a new section

`### Quickly changing between layouts`

Comment (crutkas) [inline hub/powertoys/fancyzones.md]: "With a custom layout, this layout can be ...

File: hub/powertoys/fancyzones.md [hunk 4]
```diff
@@ -64,6 +64,12 @@ The additive **Canvas** model starts with a blank layout and supports adding zon
...
+
+In the demo below, we start with a default template applied to the screen and 2 custom layouts that we assign hotkeys for. We then use the `Win ⊞ + Ctrl + Alt + [hotkey]` binding to apply the first custom layout and bind a window to it. Finally, we apply the second custom layout while dragging a window and bind the window to it.
```
Comment (crutkas) [inline hub/powertoys/fancyzones.md:71]: @mattwojo does docs have a 'description' helper block for images?

**Review by crutkas — APPROVED** [review]
Good but suggest a few items

**Comment by htcfreek** [comment]
@dedavis6797 , @crutkas 
Can we add a big warning for the comming support end on older than 1903?

**Comment by mattwojo** [comment]
We've got a warning block you can use, syntax is:
> [!WARNING]
> Support ending for any versions older than 1903 or whatever you want to say.

https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=master#alerts-note-tip-important-caution-warning

**Comment by crutkas** [comment]
@mattwojo good link, @dedavis6797 we should be using these more.  https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=master#alerts-note-tip-important-caution-warning

**Comment by mattwojo** [comment]
Just beware that we've also found that some readers just skip over the Notes/Important/Warning blocks... I think the key is to use them somewhat sparingly.

**Comment by PRMerger6** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by dedavis6797** [comment]
With these last commits I think the Docs are in good shape for v0.35. @mattwojo can you take a look and help get this merged?

**Comment by htcfreek** [comment]
@dedavis6797 
The settings section on FZ page needs a big overhaul/rework. I suggest to do this the next days or with 0.37.

For other things you get a 👍 from me.

cc: @crutkas, @enricogior

**Pull request merged**